services/app-api/forms/2026/elements.ts

```diff
@@ -392,3 +393,19 @@ export const performanceRateTermStay: PerformanceRateTemplate = {
   required: true,
   multiplier: 1000,
 };
+
+export const sectionCompleteBanner: StatusAlertTemplate = {
+  type: ElementType.StatusAlert,
+  id: "status-alert",
+  title: "Measure has been completed",
+  text: "You can still edit the this section but the the status will change to ‘In progress’ and you will need to re-select the ‘Complete section button. Click here or select ‘Return to measure details’ button above to return to the previous page.",
```

Comment:
shmoll typos and questions: 
- Should it be "the" or "this"
    >You can still edit the this section...
- Do we want to add a closing ` to "Complete" in this:
    >... re-select the ‘Complete section button.
- For this part, should we say "link" instead of "button" 
    >...select ‘Return to measure details’ button above

Reply:
all valid questions, this can be handled in a follow-on content PR, along with the link